### DISCUSSION TITLE
How to fix color issues with Mermaid?

### DISCUSSION THREAD

**Discussion by Sun-ZhenXing** [vuepress-theme-hope/vuepress-theme-hope#3433] [Q&A] [answered]
There are some color issues with the `erDiagram` in night mode.

<img width="267" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/44517244/0f23d117-d29b-490f-a010-605ddc3a8b96">

Test code:

````markdown
```mermaid
erDiagram
    Category ||--o{ Category: "类目是树形结构"
    Category ||--o{ Shop: "每个店铺都在一个一级类目中"

    Category {
        int id PK "ID"
        int pid FK "父类目 ID"
        string name "类目名称"
        int star "星级"
        string remark "备注"
        bool is_deleted "是否删除"
        int version "乐观锁"
        datetime created_at "创建时间"
        datetime updated_at "更新时间"
    }
```
````

**Comment by Mister-Hope** [comment]
I think at least the demo is working, and I think I override all color variables. I will check that later

**Reply by Mister-Hope** [reply]
I am not overriding all variables, and I have made several prs upstream, this shall be fixed later

**Reply by Mister-Hope** [reply]
https://github.com/mermaid-js/mermaid/issues/4931

**Reply by Sun-ZhenXing** [reply]
Thank you for your feedback.

**Comment by Mister-Hope** [comment, marked as answer]
Fixed in https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/7279e7c8f8904fb4bfd0745e7f20837483747560
